### PR TITLE
Generate ACH Transfer Receipt/Confirmation Letter

### PR DESCRIPTION
**Pull request by garyhtou**
#1191

### CODE REVIEW

**Comment by garyhtou** [comment]
Getting wkhtmltopdf error. I tried installing, but no luck.

image

### My dev env setup:

Docker running on WSL2 Ubuntu 20.04.2 (Windows 10 Home)

**Comment by motdotla** [comment]
what version of `pdftoppm` do you have?

```
❯ pdftoppm -v                                                                                                                                                                                                                                        14:42:38
pdftoppm version 21.02.0
Copyright 2005-2021 The Poppler Developers - http://poppler.freedesktop.org
Copyright 1996-2011 Glyph & Cog, LLC
```

**Comment by motdotla** [comment]
nevermind my last comment. that is for png generation.

**Comment by motdotla** [comment]
Try pulling the latest, running bundle install, and trying to generate again. I added the binary for development mode.

File: app/views/ach_transfers/transfer_confirmation_letter.pdf.erb [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      <p>
+        Please don't hesitate to contact me with any additional questions. I can be reached at <a href="mailto:michael@hackclub.com">michael@hackclub.com</a>
+      </p>
+    </div>
+
+    <br />
+    <p>
+      Best,
+    </p>
+    <%= image_tag(wicked_pdf_asset_base64('michael_signature.png'), width: 300, height: 73, dpi: 96) %>
```
Comment (motdotla) [inline app/views/ach_transfers/transfer_confirmation_letter.pdf.erb]: This should now be melanie_signature (or something close to that. double check the image was added after doing a rebase)

File: app/views/ach_transfers/transfer_confirmation_letter.pdf.erb [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    <p style="line-height: 1.5">
+      Michael Destefanis<br />
```
Comment (motdotla) [inline app/views/ach_transfers/transfer_confirmation_letter.pdf.erb]: same here

File: app/views/ach_transfers/transfer_confirmation_letter.pdf.erb [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+      <p><%= @ach_transfer.status %></p>
+      <p>
+        This letter is to verify that EVENT NAME blah blah is acting as the nonprofit <a href="https://en.wikipedia.org/wiki/Fiscal_sponsorship">fiscal sponsor</a> for <%= @event.name %>.
+      </p>
+      <p>
+        We have established a restricted fund within our foundation to support their event and are
+receiving funds on their behalf. The Hack Foundation is a 501(c)(3) public charity with the EIN 81-2908499.
+      </p>
+      <p>
+        Please don't hesitate to contact me with any additional questions. I can be reached at <a href="mailto:michael@hackclub.com">michael@hackclub.com</a>
```
Comment (motdotla) [inline app/views/ach_transfers/transfer_confirmation_letter.pdf.erb]: https://github.com/hackclub/bank/pull/1374/files

**Review by motdotla — CHANGES_REQUESTED** [review]

**Comment by garyhtou** [comment]
I'm surprised this is working lmao. Let me if there can be any improvements

hm... idk why the rebase created duplicate commits

**Comment by garyhtou** [comment]
[transfer_confirmation_letter (1).pdf](https://github.com/hackclub/bank/files/6256709/transfer_confirmation_letter.1.pdf)

**Review by motdotla — APPROVED** [review]
good work here Gary

**Pull request merged**